catalog/CSIC/CSIC_SPEI_2_8.jsonnet

```diff
@@ -0,0 +1,330 @@
+      },
+      {
+        name: 'SPEI_36_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 36 months',
+      },
+      {
+        name: 'SPEI_37_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 37 months',
+      },{
+        name: 'SPEI_38_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 38 months',
+      },
+      {
+        name: 'SPEI_39_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 39 months',
+      },
+      {
+        name: 'SPEI_40_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 40 months',
+      },
+      {
+        name: 'SPEI_41_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 41 months',
+      },
+      {
+        name: 'SPEI_42_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 42 months',
+      },
+      {
+        name: 'SPEI_43_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 43 months',
+      },
+      {
+        name: 'SPEI_44_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 44 months',
+      },
+      {
+        name: 'SPEI_45_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 45 months',
+      },
+      {
+        name: 'SPEI_46_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 46 months',
+      },
+      {
+        name: 'SPEI_47_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 47 months',
+      },
+      {
+        name: 'SPEI_48_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 48 months',
+      },
+    ],
+    'gee:visualizations': [
```

Comment:
Tighten up the `gee:visualization` so that it is more readable.  See for example here:

https://github.com/google/earthengine-catalog/blob/0df66304a8fe4f8ff28589c74eff186a358f00a1/catalog/UQ/UQ_murray_Intertidal_v1_1_data_mask.jsonnet#L88